src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1063,33 +1065,32 @@ public LocalDate fromString(String string) {
         case STANDARD -> {
           textFieldMaxRankingsAllowed.setDisable(false);
           textFieldMinimumVoteThreshold.setDisable(false);
-          choiceTiebreakMode.setDisable(false);
-          checkBoxHareQuota.setDisable(false);
           checkBoxBatchElimination.setDisable(false);
           checkBoxContinueUntilTwoCandidatesRemain.setDisable(false);
+          choiceTiebreakMode.setDisable(false);
+          textFieldNumberOfWinners.setText("1");
         }
         case MULTI_SEAT_ALLOW_ONLY_ONE_WINNER_PER_ROUND, MULTI_SEAT_ALLOW_MULTIPLE_WINNERS_PER_ROUND -> {
           textFieldMaxRankingsAllowed.setDisable(false);
           textFieldMinimumVoteThreshold.setDisable(false);
           choiceTiebreakMode.setDisable(false);
-          checkBoxNonIntegerWinningThreshold.setDisable(false);
-          checkBoxHareQuota.setDisable(false);
+          radioThresholdMostCommon.setDisable(false);
+          radioThresholdHbQuota.setDisable(false);
+          radioThresholdHareQuota.setDisable(false);
           textFieldDecimalPlacesForVoteArithmetic.setDisable(false);
           textFieldNumberOfWinners.setDisable(false);
         }
         case MULTI_SEAT_BOTTOMS_UP_UNTIL_N_WINNERS, MULTI_SEAT_SEQUENTIAL_WINNER_TAKES_ALL -> {
           textFieldMaxRankingsAllowed.setDisable(false);
           textFieldMinimumVoteThreshold.setDisable(false);
           choiceTiebreakMode.setDisable(false);
-          checkBoxHareQuota.setDisable(false);
           textFieldNumberOfWinners.setDisable(false);
         }
         case MULTI_SEAT_BOTTOMS_UP_USING_PERCENTAGE_THRESHOLD -> {
           textFieldMaxRankingsAllowed.setDisable(false);
           textFieldMinimumVoteThreshold.setDisable(false);
           choiceTiebreakMode.setDisable(false);
-          checkBoxHareQuota.setDisable(false);
-          textFieldNumberOfWinners.setDisable(false);
+          textFieldNumberOfWinners.setText("0");
```

Comment:
can we hide it entirely in this case?

Reply:
The answer is yes, but it wouldn't move any of the options around, it'd just appear as a blank space there, which is awkward. I also think we've had this conversation a number of times and I've been pretty firmly in the camp of keeping the user as aware of what ends up in the config file as possible. Happy to reassess this in the future for the entire GUI if you'd like to file an issue for that. Chris would probably be excited about this idea.